src/SubjectPreview.js

```diff
@@ -77,12 +79,22 @@ SubjectPreview.propTypes = {
         // day: PropTypes.string,
         // time: PropTypes.string,
     }),
-
     /** Event callback - user clicked on subject name and wants more info */
     onShowMore: PropTypes.func,
+    /** Render the subject as selected (with highlight) */
+    selected: PropTypes.bool,
+    /** Event callback - user wants to de/select this subject */
+    onToggleSelect: PropTypes.func,
 };
 
-export default SubjectPreview;
+export default connect(
+    ({selectedSubjects}, {subject}) => ({
+        selected: selectedSubjects.has(subject.predmet_id)
+    }),
+    (dispatch, {subject}) => ({
```

Comment:
To jsem nezmínil. Za to se omlouvám, doufám, že tě to moc nezdrželo, ale takhle je to správně.

Reply:
Jo v pohodě, dokumentace pro ten connect je hezká.